commit cc464ed5cea27386a2e077724941cc35faf78dc5 (refs/changes/42/1842/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-25T16:28:24-07:00 (2 years, 5 months ago)
    
    IMGAPI-623 `updates-imgadm -d ...` overly verbose: with node 0.10 it is serializing full "res" field (bump ver for a release)

diff --git a/CHANGES.md b/CHANGES.md
index fca1665..92b3a0c 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,10 @@
 
 ## not yet released
 
+## 2.2.5
+
+- IMGAPI-623 `updates-imgadm -d ...` overly verbose
+
 ## 2.2.4
 
 - IMGAPI-606 Fix '--identity' (and the equiv `*_IMGADM_IDENTITY` env var) to
diff --git a/package.json b/package.json
index 0373f43..c793787 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi-cli",
   "description": "a CLI for an Triton's IMGAPI (https://images.joyent.com/docs/)",
-  "version": "2.2.4",
+  "version": "2.2.5",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
